src/components/UIShared/ConversationActionsShortcut.vue

```diff
@@ -0,0 +1,151 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { isNavigationFailure, NavigationFailureType } from 'vue-router'
+import { useRouter, useRoute } from 'vue-router/composables'
+
+import IconCheckUnderline from 'vue-material-design-icons/CheckUnderline.vue'
+import IconDelete from 'vue-material-design-icons/Delete.vue'
+
+import { showError } from '@nextcloud/dialogs'
+import { t, getLanguage } from '@nextcloud/l10n'
+
+import NcButton from '@nextcloud/vue/components/NcButton'
+import { spawnDialog } from '@nextcloud/vue/functions/dialog'
+
+import ConfirmDialog from '../../components/UIShared/ConfirmDialog.vue'
+
+import { useStore } from '../../composables/useStore.js'
+import { CONVERSATION } from '../../constants.ts'
+import { hasTalkFeature, getTalkConfig } from '../../services/CapabilitiesManager.ts'
+
+const supportsArchive = hasTalkFeature('local', 'archived-conversations-v2')
+const retentionEventPeriod = getTalkConfig('local', 'conversations', 'retention-event')
+const retentionPhonePeriod = getTalkConfig('local', 'conversations', 'retention-phone')
+
+const props = defineProps<{
+	token: string,
+	objectType: string,
+	isHighlighted: boolean,
+}>()
+
+const store = useStore()
+const router = useRouter()
+const route = useRoute()
+
+const isModerator = computed(() => store.getters.isModerator)
+
+const expirationDuration = computed(() => {
+	if (props.objectType === CONVERSATION.OBJECT_TYPE.EVENT) {
+		return retentionEventPeriod
+	} else if (props.objectType === CONVERSATION.OBJECT_TYPE.PHONE_TEMPORARY) {
+		return retentionPhonePeriod
+	}
+	return 0
+})
+
+const descriptionLabel = computed(() => {
+	const expirationDurationFormatted = new Intl.RelativeTimeFormat(getLanguage(), { numeric: 'always' }).format(
+		expirationDuration.value, 'days'
+	)
+	return t('spreed', 'This conversation will be automatically deleted for everyone {expirationDurationFormatted} of no activity.', { expirationDurationFormatted })
```

Comment:
0 means disabled retention, we need a different message in that case.

Reply:
But still should show buttons? Currently they just hidden

Reply:
I would still offer the delete now and keep.
Just in case admin changes their mind later

Reply:
for `expirationDuration === 0`, we should still show these actions for moderator to act (intention is to clean up after the call, no matter if it's done by background job or manually)
- for both event and phone rooms could be `The call has ended. Would you like to delete this conversation?`